### PR TITLE
Icons: Implemented chevron, left arrow and provile icons

### PR DESCRIPTION
**Pull request by Dulaya**
Created three icons from AntDesign and MaterialCommunityIcons: chevron, left arrow and profile icons.

### CODE REVIEW

**Comment by Dulaya** [comment]
image

**Review by Nap1400 — APPROVED** [review]

**Review by carizzuti — APPROVED** [review]
Images and sizes look good

**Review by raynesmike — APPROVED** [review]
Looks good.

File: src/components/index.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ export * from './container/Container';
 export * from './listView/ListView';
 export * from './button/Button';
 export * from './icon/Icon';
```
Comment (lasharela) [inline src/components/index.js:5]: icon / icons?

**Review by lasharela — APPROVED** [review]

**Pull request merged**